rsocket/RSocket.h

```diff
@@ -19,7 +21,7 @@ class RSocket {
       std::shared_ptr<RSocketResponder> responder =
           std::make_shared<RSocketResponder>(),
       std::unique_ptr<KeepaliveTimer> keepaliveTimer =
```

Comment:
Should we just change this to take `folly::Optional<std::chrono::milliseconds>`? It's unlikely we will ever have a different `KeepAlive` impl and this lets us setting the EventBase below.